Hunting Queries/Corelight/CorelightFilesTransferedByIp.yaml

```diff
@@ -0,0 +1,28 @@
+id: 887e648f-32d7-4803-8cd4-2ceef7995872
+name: Corelight - File transferred by source
+description: |
+  'Query searches for files transferred over period of time by each source.'
+severity: Low
+requiredDataConnectors:
+  - connectorId: Corelight
+    dataTypes:
+      - Corelight_CL

```

Comment:
If we are using parser in query datatype should be parser name. So could you please update the datatype name to Corelight.

Reply:
updated